codebuild/common-windows.bat

```diff
@@ -1,14 +1,15 @@
 
-set CMAKE_ARGS=%*
 
 @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
-choco install %PYTHON_PACKAGE% -y
+echo "Installing python version: %* via choco"
+choco install %* -y
+dir "C:\Program Files (x86)\Microsoft Visual Studio"
```

Comment:
is this `dir` just for debugging?
do you still want it here?

Reply:
removed now that I know why VS2017 wasn't working.